sql/structured.proto

```diff
@@ -73,6 +73,13 @@ message IndexDescriptor {
   // parallels the column_names list.
   repeated uint32 column_ids = 5 [(gogoproto.customname) = "ColumnIDs",
       (gogoproto.casttype) = "ColumnID"];
+  // An ordered list of additional column ids to use for the index. For
+  // non-unique indexes, these columns will be appended to the key. For unique
+  // indexes these columns will be stored in the value. The extra column IDs is
+  // computed as PrimaryIndex.column_ids - column_ids. For the primary index
+  // the list will be empty.
+  repeated uint32 extra_column_ids = 6 [(gogoproto.customname) = "ExtraColumnIDs",
```

Comment:
I renamed to `ImplicitColumnIDs` as I find that the most palatable. These column IDs are implicit in the sense that they were automatically associated with the index. `Disambiguating` feels wrong to me because disambiguation is not needed for unique indexes.